src/unlimited.cpp

```diff
@@ -1153,7 +1153,16 @@ bool IsChainNearlySyncd()
     LOCK(cs_ischainnearlysyncd);
     return fIsChainNearlySyncd;
 }
-
+uint64_t LargestBlockSeen(uint64_t nBlockSize)
+{
+     // return the largest block size that we have seen since startup
+     static uint64_t nLargestBlockSeen = BLOCKSTREAM_CORE_MAX_BLOCK_SIZE;
```

Comment:
This is the block size which is preferred by Blockstream and Core at the moment. It is neither unthruthful nor defamatory, simply accurate.